app/templates/auth/request-password-reset.html

```diff
@@ -3,23 +3,19 @@
 {% block page_title %}Reset password – Digital Marketplace{% endblock %}
 
 {% block breadcrumb %}
-  {%
-    with items = [
-      {
-        "link": "/",
-        "label": "Digital Marketplace"
-      },
-      {
-        "link": "/user/login",
-        "label": "Login"
-      },
-      {
-        "label": "Reset password"
-      }
-    ]
-  %}
-    {% include "toolkit/breadcrumb.html" %}
-  {% endwith %}
+  {{ govukBreadcrumbs({"items": [
+    {
+      "text": "Digital Marketplace",
+      "href": "/",
+    },
+    {
+      "text": "Login",
+      "href": "/user/login",
```

Comment:
As part of the password reset user journey why would I need to go back to the user login page? 
Would I be able to authenticate again? would it invalidate my existing session?